CONTRIBUTING.rst

```diff
@@ -63,8 +67,9 @@ Contributing
 
 
 **Step 5**: Now it's probably time to check the unit tests and:
- * make sure unit tests for the modules you touched are still succeeding
- * create new unit test(s)
+
+- make sure unit tests for the modules you touched are still succeeding
+- create new unit test(s)
```

Comment:
I think something should be said or referenced regarding the unit test structure expected when creating new tests.
The standards directory README file describes that well:

https://github.com/dmwm/WMCore/blob/master/standards/README#L78-L96

Reply:
That's a good point! Which actually brings us to multiple documentation on how to contribute to the project. It's probably better to either:
* update the README file
* or to integrate it into the contribution guidelines and completely get rid of README

Let me check what would be the best path forward here.